app/models/login_link.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class LoginLink
+  def self.create!(email)
+    auth_code = AuthCode.create!
+
+    url = Rails.application.routes.url_helpers.verify_email_url(
```

Comment:
This is very expensive since every call it will create all the routes helpers again. You could cache in a constant, or I think use `AppplicationController.helpers`

Reply:
Good call. On the next part we can benchmark this and see what's the best solution.

https://github.com/fnando/35mm/issues/3